### PR TITLE
update docs on supported AWS Control Tower integration CU-n18r68

### PR DESCRIPTION
**Pull request by j0g3sc**
This is related to https://github.com/meshcloud/meshfed-release/pull/1474

### CODE REVIEW

**Comment by meshcloud-gmbh** [comment]
Task linked: [CU-n18r68 Implement EnrollAccountIntoControlTowerStep for Replicator](https://app.clickup.com/t/n18r68)

File: docs/meshstack.aws.landing-zones.md [hunk 1]
```diff
@@ -136,7 +138,30 @@ The following parameters can be used in the Lambda function and are provided as
 In addition, any payment settings, project tags or customer tags can also be used as Lambda function parameters. These parameter keys will have the prefix `tag`.
 For example, the value of the tag `costCenter` will be made available via the key `tagCostCenter`.
 
-If you would like to downgrade the permissions that meshStack recieved in the newly provisioned account, you can do that inside the same Lambda. See the [Downgrading meshCloud Access](./meshstack.aws.index.md#downgrading-meshcloud-access) section for an example.
+If you would like to downgrade the permissions that meshStack received in the newly provisioned account, you can do that inside the same Lambda. See the [Downgrading meshCloud Access](./meshstack.aws.index.md#downgrading-meshcloud-access) section for an example.
```
Comment (Jelledb) [inline docs/meshstack.aws.landing-zones.md:141]:
```suggestion
If you would like to downgrade the permissions that meshStack received in the newly provisioned account, you can do that inside the same Lambda. See the [Downgrading meshcloud Access](./meshstack.aws.index.md#downgrading-meshcloud-access) section for an example.
```

File: docs/meshstack.aws.landing-zones.md [hunk 2]
```diff
@@ -136,7 +138,30 @@ The following parameters can be used in the Lambda function and are provided as
...
+> In order to enroll created accounts with AWS Control Tower, the [Target Organization Unit ID](#target-organization-unit-id) must belong to a OU that is already enrolled.
+
+Following prerequisites must be fulfilled for the enrollment to work:
+
+- Configured AWS Control Tower
+- AWS Service Catalog needs to have an AWS Control Tower Account Factory Portfolio
+- The portfolio needs to contain an AWS Control Tower Account Factory Product
+- The AWS Control Tower Account Factory Product needs to have at least one active Provisioned Artifact (active version)
+- There needs to be a launch path with permissions to invoke the AWS Account Factory Product for meshStack access
+- all of [AWS prerequisites](https://docs.aws.amazon.com/controltower/latest/userguide/enroll-account.html)
```
Comment (Jelledb) [inline docs/meshstack.aws.landing-zones.md:164]:
```suggestion
- All [AWS prerequisites](https://docs.aws.amazon.com/controltower/latest/userguide/enroll-account.html) for enrolling accounts
```

**Review by Jelledb — APPROVED** [review]

**Comment by j0g3sc** [comment]
Re-requested review because documentation changed completely.

File: docs/meshstack.aws.index.md [hunk 3]
```diff
@@ -307,6 +307,38 @@ customer_identifier_length: 16
...
+
+The enrollment configuration looks like this:
+
+```dhall
+{ management-account-id : Text
+, account-factory-product-id : Text
+}
+```
+
+Following prerequisites must be fulfilled for the enrollment to work:
```
Comment (Jelledb) [inline docs/meshstack.aws.index.md:328]:
```suggestion
The following prerequisites must be fulfilled for the enrollment to work:
```

File: docs/meshstack.aws.index.md [hunk 4]
```diff
@@ -687,4 +720,4 @@ def main(event, context):
 
 This Lambda will be invoked during account provisioning and downgrades the `MeshstackAccountAccessRole`.
 
-> Depending of your mode of operation (usage of external Account Vending Machine) these "minimal rights" can be adapted and further restricted. Please [contact us](https://support.meshcloud.io) for more details on reducing these rights.
+> Depending of your mode of operation (usage of external Account Vending Machine or configuration of AWS Control Tower enrollment) these "minimal rights" can be adapted and further restricted. Please [contact us](https://support.meshcloud.io) for more details on reducing these rights.
```
Comment (Jelledb) [inline docs/meshstack.aws.index.md:723]:
```suggestion
> Depending of your mode of operation (usage of external Account Vending Machine or configuration of AWS Control Tower enrollment) these "minimal rights" can be adapted and further restricted. Please [contact us](mailto:support@meshcloud.io) for more details on reducing these rights.
```
Not even sure if a mailto links works here, please test it. But `support.meshcloud.io` doesn't lead anywhere ;-)

**Review by Jelledb — APPROVED** [review]

File: docs/meshstack.aws.index.md (hunk 4 above)
Comment (j0g3sc) [reply docs/meshstack.aws.index.md:723]: Currently this does not work so i will go with the mailto link, that should work i guess.

**Pull request merged**